### PR TITLE
Fix tx fee summary display

### PR DESCRIPTION
**Pull request by SehKeat**
Fix tx fee summary display for create transfer, namespace and asset.

### CODE REVIEW

File: src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue [hunk 1]
```diff
@@ -400,6 +424,16 @@ export default {
         cosignerAddress.value = n.cosignerList.length>0?getMultiSigCosigner.value.cosignerList[0].address:''
       }
     })
+
+    const findAccWithAddress = address =>{
+      return walletState.currentLoggedInWallet.accounts.find(acc=>acc.address==address)
+    }
+
```
Comment (anakin001) [inline src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue:431]: walletState.currentLoggedInWallet can be null, the app will hang when user clicks log out, please return null if it's null

File: src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue [hunk 2]
```diff
@@ -400,6 +424,16 @@ export default {
...
+    const checkCosignBalance = computed(() => {
+      let cosignBalance = findAccWithAddress(cosignerAddress.value).balance;
+      return cosignBalance;
+    })
+
```
Comment (anakin001) [inline src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue:436]: same as above, findAccWithAddress can be null, return 0 if it's null

File: src/modules/transfer/views/ViewTransferCreate.vue [hunk 3]
```diff
@@ -504,9 +533,9 @@ export default {
       }
     }else {
       if(tokenDivisibility== 0){
-        return Math.trunc((checkIsNaN(parseFloat(sendXPX.value.replace(/,/g, ''))) + parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value))
+        return Math.trunc((parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value))
       }else{
-        return Math.round((checkIsNaN(parseFloat(sendXPX.value.replace(/,/g, ''))) + parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value)*Math.pow(10,tokenDivisibility))/ Math.pow(10,tokenDivisibility)
+        return Math.round((parseFloat(effectiveFee.value)+ lockFundTxFee.value + lockFund.value)*Math.pow(10,tokenDivisibility))/ Math.pow(10,tokenDivisibility)
       }
```
Comment (anakin001) [inline src/modules/transfer/views/ViewTransferCreate.vue:539]: why is this changed?

Comment (SehKeat) [reply src/modules/transfer/views/ViewTransferCreate.vue:539]: the sendxpx value only needed to total up if it is non multisig. If it is multisig, the sendxpx value will only show for the primary(multisig) account and the total(Aggregate Tx Fee, Lock Fund Tx Fee and Lock Fund) for the signer.

File: src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue (hunk 1 above)
Comment (SehKeat) [reply src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue:431]: noted, will revise it

File: src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue (hunk 2 above)
Comment (SehKeat) [reply src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue:436]: noted, will revise it

Comment (SehKeat) [reply src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue:442]: I have revised the code, please let me know if it done correctly. thanks

Comment (anakin001) [reply src/modules/services/submodule/assets/views/ViewServicesAssetsCreate.vue:442]: if(findAcc..) is null, .balance will be undefined

**Review by anakin001 — APPROVED** [review]
